scipy/meson.build

```diff
@@ -80,22 +80,27 @@ npymath_lib = cc.find_library('npymath', dirs: npymath_path)
 npyrandom_lib = cc.find_library('npyrandom', dirs: npyrandom_path)
 
 # pybind11 include directory - needed in several submodules
-incdir_pybind11 = run_command(py3,
-  [
-    '-c',
-    '''from os.path import relpath
+#
+# in meson 1.1.0, this will find the pyproject.toml build-requires version
+pybind11_dep = dependency('pybind11', version: '>=2.10.0', required: false, allow_fallback: true)
```

Comment:
@eli-schwartz one question on `allow_fallback: true` - that doesn't see to do anything, because we don't have a `subprojects/pybind11`. Is there a reason to add this (allow packagers to add a subproject, just good style, ...)?

Reply:
Ah okay, that makes sense, thanks.